---a/net/minecraft/world/gen/layer/GenLayerBiome.java
+++b/net/minecraft/world/gen/layer/GenLayerBiome.java
@@ -4,8 +4,8 @@
 import net.minecraft.world.biome.BiomeGenBase;
 import net.minecraft.world.gen.ChunkProviderSettings;
 
-public class GenLayerBiome extends GenLayer
-{
+public class GenLayerBiome extends GenLayer {
+
     private BiomeGenBase[] field_151623_c;
     private BiomeGenBase[] field_151621_d;
     private BiomeGenBase[] field_151622_e;
@@ -13,101 +13,66 @@
     private final ChunkProviderSettings field_175973_g;
     private static final String __OBFID = "CL_00000555";
 
-    public GenLayerBiome(long p_i45560_1_, GenLayer p_i45560_3_, WorldType p_i45560_4_, String p_i45560_5_)
-    {
+    public GenLayerBiome(long p_i45560_1_, GenLayer p_i45560_3_, WorldType p_i45560_4_, String p_i45560_5_) {
         super(p_i45560_1_);
-        this.field_151623_c = new BiomeGenBase[] {BiomeGenBase.desert, BiomeGenBase.desert, BiomeGenBase.desert, BiomeGenBase.savanna, BiomeGenBase.savanna, BiomeGenBase.plains};
-        this.field_151621_d = new BiomeGenBase[] {BiomeGenBase.forest, BiomeGenBase.roofedForest, BiomeGenBase.extremeHills, BiomeGenBase.plains, BiomeGenBase.birchForest, BiomeGenBase.swampland};
-        this.field_151622_e = new BiomeGenBase[] {BiomeGenBase.forest, BiomeGenBase.extremeHills, BiomeGenBase.taiga, BiomeGenBase.plains};
-        this.field_151620_f = new BiomeGenBase[] {BiomeGenBase.icePlains, BiomeGenBase.icePlains, BiomeGenBase.icePlains, BiomeGenBase.coldTaiga};
+        this.field_151623_c = new BiomeGenBase[]{BiomeGenBase.desert, BiomeGenBase.desert, BiomeGenBase.desert, BiomeGenBase.savanna, BiomeGenBase.savanna, BiomeGenBase.plains};
+        this.field_151621_d = new BiomeGenBase[]{BiomeGenBase.forest, BiomeGenBase.roofedForest, BiomeGenBase.extremeHills, BiomeGenBase.plains, BiomeGenBase.birchForest, BiomeGenBase.swampland};
+        this.field_151622_e = new BiomeGenBase[]{BiomeGenBase.forest, BiomeGenBase.extremeHills, BiomeGenBase.taiga, BiomeGenBase.plains};
+        this.field_151620_f = new BiomeGenBase[]{BiomeGenBase.icePlains, BiomeGenBase.icePlains, BiomeGenBase.icePlains, BiomeGenBase.coldTaiga};
         this.parent = p_i45560_3_;
 
-        if (p_i45560_4_ == WorldType.DEFAULT_1_1)
-        {
-            this.field_151623_c = new BiomeGenBase[] {BiomeGenBase.desert, BiomeGenBase.forest, BiomeGenBase.extremeHills, BiomeGenBase.swampland, BiomeGenBase.plains, BiomeGenBase.taiga};
+        if (p_i45560_4_ == WorldType.DEFAULT_1_1) {
+            this.field_151623_c = new BiomeGenBase[]{BiomeGenBase.desert, BiomeGenBase.forest, BiomeGenBase.extremeHills, BiomeGenBase.swampland, BiomeGenBase.plains, BiomeGenBase.taiga};
             this.field_175973_g = null;
-        }
-        else if (p_i45560_4_ == WorldType.CUSTOMIZED)
-        {
+        } else if (p_i45560_4_ == WorldType.CUSTOMIZED) {
             this.field_175973_g = ChunkProviderSettings.Factory.func_177865_a(p_i45560_5_).func_177864_b();
-        }
-        else
-        {
+        } else {
             this.field_175973_g = null;
         }
     }
 
-    public int[] getInts(int areaX, int areaY, int areaWidth, int areaHeight)
-    {
+    public int[] getInts(int areaX, int areaY, int areaWidth, int areaHeight) {
         int[] aint = this.parent.getInts(areaX, areaY, areaWidth, areaHeight);
         int[] aint1 = IntCache.getIntCache(areaWidth * areaHeight);
 
-        for (int i1 = 0; i1 < areaHeight; ++i1)
-        {
-            for (int j1 = 0; j1 < areaWidth; ++j1)
-            {
-                this.initChunkSeed((long)(j1 + areaX), (long)(i1 + areaY));
+        for (int i1 = 0; i1 < areaHeight; ++i1) {
+            for (int j1 = 0; j1 < areaWidth; ++j1) {
+                this.initChunkSeed((long) (j1 + areaX), (long) (i1 + areaY));
                 int k1 = aint[j1 + i1 * areaWidth];
                 int l1 = (k1 & 3840) >> 8;
                 k1 &= -3841;
 
-                if (this.field_175973_g != null && this.field_175973_g.fixedBiome >= 0)
-                {
+                if (this.field_175973_g != null && this.field_175973_g.fixedBiome >= 0) {
                     aint1[j1 + i1 * areaWidth] = this.field_175973_g.fixedBiome;
-                }
-                else if (isBiomeOceanic(k1))
-                {
+                } else if (isBiomeOceanic(k1)) {
                     aint1[j1 + i1 * areaWidth] = k1;
-                }
-                else if (k1 == BiomeGenBase.mushroomIsland.biomeID)
-                {
+                } else if (k1 == BiomeGenBase.mushroomIsland.biomeID) {
                     aint1[j1 + i1 * areaWidth] = k1;
-                }
-                else if (k1 == 1)
-                {
-                    if (l1 > 0)
-                    {
-                        if (this.nextInt(3) == 0)
-                        {
+                } else if (k1 == 1) {
+                    if (l1 > 0) {
+                        if (this.nextInt(3) == 0) {
                             aint1[j1 + i1 * areaWidth] = BiomeGenBase.mesaPlateau.biomeID;
-                        }
-                        else
-                        {
+                        } else {
                             aint1[j1 + i1 * areaWidth] = BiomeGenBase.mesaPlateau_F.biomeID;
                         }
-                    }
-                    else
-                    {
+                    } else {
                         aint1[j1 + i1 * areaWidth] = this.field_151623_c[this.nextInt(this.field_151623_c.length)].biomeID;
                     }
-                }
-                else if (k1 == 2)
-                {
-                    if (l1 > 0)
-                    {
+                } else if (k1 == 2) {
+                    if (l1 > 0) {
                         aint1[j1 + i1 * areaWidth] = BiomeGenBase.jungle.biomeID;
-                    }
-                    else
-                    {
+                    } else {
                         aint1[j1 + i1 * areaWidth] = this.field_151621_d[this.nextInt(this.field_151621_d.length)].biomeID;
                     }
-                }
-                else if (k1 == 3)
-                {
-                    if (l1 > 0)
-                    {
+                } else if (k1 == 3) {
+                    if (l1 > 0) {
                         aint1[j1 + i1 * areaWidth] = BiomeGenBase.megaTaiga.biomeID;
-                    }
-                    else
-                    {
+                    } else {
                         aint1[j1 + i1 * areaWidth] = this.field_151622_e[this.nextInt(this.field_151622_e.length)].biomeID;
                     }
-                }
-                else if (k1 == 4)
-                {
+                } else if (k1 == 4) {
                     aint1[j1 + i1 * areaWidth] = this.field_151620_f[this.nextInt(this.field_151620_f.length)].biomeID;
-                }
-                else
-                {
+                } else {
                     aint1[j1 + i1 * areaWidth] = BiomeGenBase.mushroomIsland.biomeID;
                 }
             }
@@ -116,4 +116,4 @@
 
         return aint1;
     }
-}
+}
